src/main/java/com/denizenscript/denizencore/scripts/commands/core/DrawImageCommand.java

```diff
@@ -0,0 +1,65 @@
+package com.denizenscript.denizencore.scripts.commands.core;
+
+import com.denizenscript.denizencore.objects.core.ImageTag;
+import com.denizenscript.denizencore.scripts.commands.AbstractCommand;
+import com.denizenscript.denizencore.scripts.commands.generator.ArgDefaultText;
+import com.denizenscript.denizencore.scripts.commands.generator.ArgName;
+import com.denizenscript.denizencore.scripts.commands.generator.ArgPrefixed;
+
+import java.awt.*;
+
+public class DrawImageCommand extends AbstractCommand {
+
+    // <--[command]
+    // @Name DrawImage
+    // @Syntax drawimage [id:<id>] [draw:<image>] [x:<x>] [y:<y>] (width:<width>) (height:<height>)
```

Comment:
This command looks like it'd be in cleanly inside of `draw`?

Reply:
Merged, lmk if the syntax looks good